src/python/pants/option/options.py

```diff
@@ -481,6 +482,12 @@ def log(log_func):
                         You can use the global native_options_validation option
                         ({doc_url('reference/global-options#native_options_validation')}) to
                         configure this check.
+
+                        Note that there is a known issue with differences in the handling of backslash
+                        escapes in config values of type list-of-string. The solution to this issue will
+                        be to change the escaping in your config values appropriately when switching to 
+                        2.23.x Typically this will mean removing superfluous escapes, and the new behavior
```

Comment:
```suggestion
                        2.23.x. Typically this will mean removing superfluous escapes, and the new behavior
```

Just adding a missing full stop.